pkg/server/storage/redpanda/like_consumer.go

```
@@ -177,26 +178,26 @@ func batchUpdateCommentLikes(deltas []*likeEventDelta) error {
 	return pgsql.DB.Transaction(func(tx *gorm.DB) error {
 		for _, d := range deltas {
 			if d.Amount > 0 {
-				var existing model.CommentLike
+				var existing commentdomain.CommentLike
 				err := tx.Where("user_id = ? AND comment_id = ?", d.UserID, d.TargetID).First(&existing).Error
 				if err == gorm.ErrRecordNotFound {
```

golangci-lint (2.12.2)

[error] QF1003: could use tagged switch on err

(staticcheck)